handbook/people-ops/hiring/job_description_guidelines.md

```diff
@@ -0,0 +1,50 @@
+# Job description template and guidelines
```

Comment:
I left a few comments in the GDoc on this page! @withdavidli should I leave them here instead?

Reply:
Since that file was already previously merged into the interviews repo, can we address any changes to this file in a separate PR?

Reply:
Oh I had no idea this was merged already. @withdavidli created a google doc to discuss this just a couple of days ago! Happy to do this in a follow-up.

Reply:
Yeah, this PR removes it from the interviews repo: https://github.com/sourcegraph/interviews/pull/145